uwblueprint/sistering
# Exit edit mode when selecting a new shift time

## Implementation description
This commit focuses on lifting the state value isEditing from the child component AdminScheduleVolunteerTable to the side panel so that when a new shift is selected, it will exit edit mode.



## Steps to test
1. Go to http://localhost:3000/admin/schedule/posting/1
2. Click "Edit"
3. The button should now say "Save"
4. Change the shift time using the dropdown
5. Check that the button goes back to saying "Edit"


## What should reviewers focus on?
* 


## Checklist
